Dockerfile.ubuntu

```diff
@@ -234,7 +296,8 @@ RUN if [ "$CHECK_COVERAGE" == "1" ] ; then true ; else bazel test ${debug_bazel_
 RUN bazel build ${debug_bazel_flags} ${minitrace_flags} --jobs $JOBS //src:ovms
 RUN bazel build ${debug_bazel_flags} --jobs $JOBS //src:libsampleloader.so
 
-RUN cd /ovms/src/example/SampleCpuExtension/ && make
+RUN if [ "$ov_use_binary" == "1" ] ; then true ; else exit 0 ; fi ; cd /ovms/src/example/SampleCpuExtension/ && OPENVINO_PATH=/opt/intel/openvino_2022 make
```

Comment:
is there need to use 2 different paths:

/opt/intel/openvino
&
/opt/intel/openvino_2022